frontend/src/app/application-forms/application-noncommercial-group/noncommercial-learn-more.component.html

```diff
@@ -14,10 +14,10 @@ <h1 id="skip-nav-target" tabindex="-1">Noncommercial group use information</h1>
         <app-section-heading [heading]="item.sectionName" [elementId]="util.createId(item.sectionName)"></app-section-heading>
         <div [innerHtml]="item.sectionCopy"></div>
       </ng-container>
-      <div>
-        <a routerLink="/" class="usa-button">Back to permits</a>
-        <a routerLink="/applications/noncommercial-group-use/new" class="usa-button-inverse usa-button">Noncommercial permit</a>
-      </div>
+
+      <app-button link="/applications/noncommercial-group-use/new" buttonText="Noncommercial permit" btnAlt="true"></app-button>
+      <hr class="light margin-top-1 margin-bottom-1">
+      <a routerLink="/" class="display-block">Back to permits</a>
```

Comment:
What do you think of making it less specific rather than more? To me it isn't clear that `permits` or `list of permits` is actually the same thing as the homepage. When I first clicked the `Back to permits` link I actually thought I would be brought somewhere other than the homepage. Perhaps more something along the lines of `< Go back`, `Return to previous page`, or `Return to homepage`?

Reply:
how about `Return to special uses homepage`

Reply:
I guess this gets at a potentially larger question - how will this app be referred to with end users? `special uses` sounds very much like an internal FS name - is `Open Forests` the public facing name of the app? If so, it's sort of a mouthful, but maybe `Return to Open Forests homepage` would be the most accurate?

Reply:
Open Forest is the name and Special Uses is a term commonly used outside the FS as well. Return to Open Forest homepage sounds better.

Reply:
just a note, that at this point that isn't the homepage for the whole platform, but we can revise when we make updates to our information architecture.